src/components/MovieDetail/MovieDetail.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
+import './MovieDetail.css';
 
 function MovieDetail({ movie, returnToMain }) {
   return (
-    <div className='movieDetail'>
-      <button onClick={()=>returnToMain()}>Go Back</button>
-      <h1>{movie.title}</h1>
-      <h2>{movie.release_date}</h2>
-      <h2>{movie.average_rating}</h2>
-      <img src={movie.poster_path}/>
-      <img src={movie.backdrop_path}/>
-
+    <div
+      className='movieDetail'
+      style={{
+        backgroundImage: `url(${movie.backdrop_path})`
+      }}
+    >
+      <div className='gradient'>
+        <button
+          className='view-all-movies-button'
+          onClick={() => returnToMain()}
+        >
+          View All Movies
+        </button>
+        {/* <img className='back-drop' src={movie.backdrop_path} /> */}
+        <section className='title-poster'>
+          <img className='poster' src={movie.poster_path} />
+          <div className='text-wrapper'>
+            <h1>{movie.title}</h1>
+            <h2 className='release-date'>Released: {movie.release_date}</h2>
+            <h2 className='rating'>Average rating: {movie.average_rating}</h2>
+            <h2 className='overview'> {movie.overview}</h2>
+          </div>
+        </section>
+      </div>
```

Comment:
You've got a lot of `move.something` - maybe on the first line of render, we can pull the keys out of `movie` using destructuring? it would save us having to type movie again and again.